test/e2e/test_cluster_configs/base.json

```diff
@@ -2,6 +2,7 @@
 	"env": {
 		"SCALE_CLUSTER": true,
 		"UPGRADE_CLUSTER": true,
+		"GET_CLUSTER_LOGS": true,
```

Comment:
If this works for Windows can we add a windows profile?

Reply:
+1